PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **Desired System State Is Declarative**
```

Comment:
```suggestion
1. **Desired System State is Declarative**
```
Typographic Suggestion:
`Is` -> `is`
to align with
https://user-images.githubusercontent.com/407675/126000898-eb81abd1-a947-40e4-b313-107fd45b7ad7.png
Thanks @scottrigby

Reply:
See https://github.com/open-gitops/documents/pull/14#discussion_r676641157